src/content/en/tools/workbox/guides/migrate.md

```diff
@@ -0,0 +1,224 @@
+path to the Workbox family of libraries. Upgrading to Workbox will provide a modern, extensible
+service worker experience with improved debugging and developer ergonomics.
+
+## Modifications to your existing configuration
+
+Regardless of which specific build tools you're migrating from, keep in mind the following name
+changes and deprecations when putting together your equivalent Workbox configuration.
+
+### Renamed options
+
+- The `runtimeCaching` config parameter has been
+[updated](https://github.com/GoogleChrome/workbox/issues/1096#issue-278581349) to take a more
+streamlined set of options, corresponding to the underlying Workbox caching plugins.
+
+- Within `runtimeCaching`, the `fastest` handler name has been
+[renamed](https://github.com/GoogleChrome/workbox/issues/915) to `staleWhileRevalidate`, which
+provides equivalent functionality.
+
+- The `dynamicUrlToDependencies` config parameter has been renamed `templatedUrls`.
+
+- The `staticFileGlobs` config parameter has been renamed `globPatterns`.
+
+### Deprecated options
+
+- Express-style wildcard routes are [no longer
+supported](https://github.com/GoogleChrome/workbox/issues/1012). If you were using Express-style
+wildcard routes in either the `runtimeCaching` configuration or directly in `sw-toolbox`, please
+migrate to an equivalent regular expression route when using Workbox.
+
+## sw-precache migrations
+
+### From the sw-precache CLI to workbox-cli
+
+Developers using the `sw-precache` command line interface, either running the command manually or as
+part of an [`npm scripts`](https://docs.npmjs.com/misc/scripts)-based build process, will find using
+the the [`workbox-cli` module](http://npmjs.com/package/workbox-cli) to be the easiest way to
+migrate. Installing `workbox-cli` will give you access to a binary called `workbox`.
+
+While the `sw-precache` CLI
+[supported](https://github.com/GoogleChromeLabs/sw-precache#command-line-interface) configuring via
+either command line flags or a configuration file, the `workbox` CLI requires that all configuration
+options be provided in a configuration file, using [CommonJS
+`module.exports`](https://nodejs.org/api/modules.html#modules_module_exports).
+
+The `workbox` CLI supports a number of different modes (use `workbox --help` to see all of them),
+but the mode that most closely matches `sw-precache`'s functionality is `generateSW`. So a call to
+
+```
+$ sw-precache --config='sw-precache-config.js'
+```
+
+can be expressed as
+
+```
+$ workbox generateSW workbox-config.js
+```
+
+### From the sw-precache node module to the workbox-build node module
+
+Developers using the `node` API for `sw-precache`, either as part of a `gulp`/`Grunt` workflow or
+just within a custom `node` build script, can migrate by switching to the `workbox-build` `node`
+module.
+
+`workbox-build`'s `generateSW()` function most closely matches the `sw-precache` `node` module's
```

Comment:
Mixing fixed and proportional fonts in the same word is particularly ugly. Try one of thes:

The `generateSW()` function (`workbox-build`)...

or 

The `workbox-build` module's `generateSW()` function...